mobile-app/cypress/integration/functional/wallet/dex/compositeswap.spec.ts

```diff
@@ -150,6 +153,18 @@ context('Wallet - DEX - Composite Swap with balance', () => {
     cy.getByTestID('slippage_input').clear().type('25').blur().wait(100)
     cy.getByTestID('button_tolerance_submit').click()
   })
+
+  it('should be able to store selected slippage value in storage', () => {
+    cy.url().should('include', 'app/DEX/CompositeSwap', () => {
+      expect(localStorage.getItem('WALLET.SLIPPAGE_TOLERANCE')).to.eq('25')
+    })
+  })
+
+  it('previously saved slippage tolerance value should be 25%', () => {
+    cy.getByTestID('text_input_tokenA').type('10')
```

Comment:
use a different amount, based on previous steps, user already typed `10` before reaching this step